libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -179,6 +181,11 @@ private static PolicyManager createPolicyManager() {
         );
     }
 
+    private static @Nullable Path getUserHome() {
+        String userHome = System.getProperty("user.home");
```

Comment:
Is this possible? I thought the jdk always sets this property.

Reply:
That was my assumption, but then, if looking at javax.activation.MimetypesFileTypeMap where this is used I'm not so sure anymore.